ActindoWorkFlow/Operation/01_ManageWorkflows.md

```diff
@@ -97,28 +101,31 @@ Connect the transition to the a place to create a valid workflow.
 
 #### Procedure
 
-*Workflows > Workflows > Tab OVERVIEW > Button Add > Button CREATE*
+*Process Orchestration > Workflows > Tab OVERVIEW > Button Add > Button CREATE*
 
 ![First action](../../Assets/Screenshots/ActindoWorkFlow/Workflows/FirstAction.png "[First action]")
 
 1. Click the ![Add](../../Assets/Icons/Plus04.png "[Add]") (Add) button next to an output port of the transition.    
     A new place connected with an arc to the selected output port is displayed.
 
+    >[Info] To zoom in or out, scroll the mouse wheel. By doing it, make sure that the mouse pointer is placed on a workflow element. If necessary, change your scroll wheel settings to adjust the zoom levels.
+
     ![First place](../../Assets/Screenshots/ActindoWorkFlow/Workflows/FirstPlace.png "[First place]")
 
 2. Add further transitions by following the procedure [Create an action](#create-an-action) or connect the place with the end place as described in the following step.
 
     > [Info] A workflow must contain at least one transition and can contain an unlimited number of transitions.
 
 3. Click the place, drag it over the place to which you want to connect it and drop it.     
-    The places are merged and the new connection has been created.
+    The places are merged, and the new connection has been created.
 
 
 
 ## Edit a workflow
 
 Edit a workflow to modify elements within the workflow, to extend the workflow by adding further transitions or to reduce the workflow by removing transitions.
 
+
 ### Select a workflow
```

Comment:
Eigentlich wählt man ein workflow version aus, innerhalb eines Workflows. Vielleicht ist es sinnvoll, es hier zu korrigieren und auch in den Prerequisites? Danach ist releavnt, wenn man workflow versions vergleicht...